nibabel/freesurfer/tests/test_io.py

```diff
@@ -9,7 +9,8 @@
 
 from ...tmpdirs import InTemporaryDirectory
 
-from nose.tools import assert_true
+
+import pytest
 import numpy as np
 from numpy.testing import assert_equal, assert_raises, dec, assert_allclose, assert_array_equal
```

Comment:
So we should also change:

* `assert_equal(a, b)` to `assert a == b`
* `assert_raises` to `pytest.raises`
* `dec.skipif(predicate, msg)` to `pytest.mark.skipif(predicate, reason=msg)`

And below it uses `from ...testing`, which we're changing temporarily to `from ...testing_pytest`.